components/chart/overview.md

```diff
@@ -68,8 +68,7 @@ Basic chart and common settings/elements
 
 ![](images/overview-chart.png)
 
->tip When configuring nested properties and child elements in your chart, the inner tags will contain their parent tag name and add specifics to its end. You can see an example of this with the `ChartSeries` > `ChartSeriesLabels` tags in the above example.
-
+@[template](/_contentTemplates/chart/link-to-basics.md#configurable-nested-chart-settings)
```

Comment:
I liked the original green tip that told people to look for nested tags. Perhaps add it back (or a variation of it) at the end of the `### Configurable Nested Chart Settings ` section